android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -116,7 +116,7 @@
 import java.lang.Integer;
 
 @SuppressLint("ViewConstructor")
-class ReactExoplayerView extends FrameLayout implements
+public class ReactExoplayerView extends FrameLayout implements
```

Comment:
really needed ?

Reply:
But we need this to be public so we can import `ReactExoplayerView` in `VideoManagerModule`